ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/WrappingRuleTest.kt

```diff
@@ -134,6 +134,24 @@ internal class WrappingRuleTest {
         ).isEmpty()
     }
 
+    @Test
+    fun testLintClassImplementsListWithComments() {
+        assertThat(
+            WrappingRule().lint(
+                """
+                interface ThingOne {}
+                interface ThingTwo {}
+                class BusyClass :
+                    ThingOne, // this comment should be legal
+                    ThingTwo {
+
+                }
+
+                """.trimIndent()
+            )
+        ).isEmpty()
+    }
+
```

Comment:
As of this week, tests have to be written in a new style. See https://github.com/pinterest/ktlint/tree/master/ktlint-test/src/main/kotlin/com/pinterest/ktlint/test